client/marketing/overview/welcome-card/style.scss

```diff
@@ -0,0 +1,63 @@
+.woocommerce-marketing-overview-welcome-card {
+	position: relative;
+
+	.woocommerce-card__body {
+		display: flex;
+		justify-content: center;
+		align-items: center;
+
+		@include breakpoint( '>960px' ) {
+			padding: 32px 108px;
+		}
+	}
+
+	&__hide-button {
```

Comment:
this is tottaly a side note, but I would suggest against those type of classes (appending & directly to class name) since they reduce discovery (someone might want to change styles for `.woocommerce-marketing-overview-welcome-card__hide-button` but can’t find it because it doesn’t exist on the source code, however due to extremely long classes, this is understandable